adminutils/adminutils.py

```diff
@@ -187,31 +187,48 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         else:
             await ctx.tick()
 
-    @emoji.command(name="message", aliases=["steal"])
-    async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles: discord.Role):
+    @emoji.command(name="steal", aliases=["member", "message"])
+    async def emoji_steal(self, ctx, id: Union[discord.Message, discord.Member, discord.Emoji, discord.PartialEmoji], name: Optional[str], *roles: discord.Role):
```

Comment:
```suggestion
    @emoji.command(name="import")
    async def emoji_import(self, ctx, id: Union[discord.Message, discord.Member, discord.Emoji, discord.PartialEmoji], name: Optional[str], *roles: discord.Role):
```
I dont really like "steal" term in this context.

Also, codefactor issue (redefining `id`, please dont use builtin names (in this case `id(obj)`) for variables) needs to be fixed.

Name arg not gonna work as optional too, because, due to how `Optional` converters work in discord.py: 
> If the converter fails to parse into the specified type, the parser will skip the parameter and then either None or the specified default will be passed into the parameter instead.

And since `str` converter cant fail, this will require specifying a name in case of roles adding

E.g.: `[p]emoji import 162379234070467641 MyAwesomeRole` will create a emoji nonrestricted to role, but with name `MyAwesomeRole`